bundles/org.openhab.binding.airquality/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -4,211 +4,231 @@
-	</channel-type>
-
-	<channel-type id="observationTime" advanced="true">
-		<item-type>DateTime</item-type>
-		<label>Observation Time</label>
-		<description>Observation date and time</description>
-		<category>Observation time</category>
-		<state readOnly="true"/>
+		<label>@text/dominentChannelLabel</label>
+		<tags>
+			<tag>Status</tag>
+		</tags>
+		<state readOnly="true">
+			<options>
+				<option value="pm25">@text/pollutantPm25</option>
+				<option value="pm10">@text/pollutantPm10</option>
+				<option value="o3">@text/pollutantO3</option>
+				<option value="no2">@text/pollutantNO2</option>
+				<option value="co">@text/pollutantCO</option>
+				<option value="so2">@text/pollutantSO2</option>
+			</options>
+		</state>
 	</channel-type>
 
-	<channel-type id="temperature" advanced="true">
+	<channel-type id="dew-point" advanced="true">
 		<item-type>Number:Temperature</item-type>
-		<label>Temperature</label>
-		<description>Temperature</description>
+		<label>@text/dewPointLabel</label>
+		<description>@text/dewPointDescription</description>
 		<category>Temperature</category>
+		<tags>
+			<tag>Point</tag>
+			<tag>Temperature</tag>
+		</tags>
 		<state readOnly="true" pattern="%.1f %unit%"/>
 	</channel-type>
 
-	<channel-type id="pressure" advanced="true">
-		<item-type>Number:Pressure</item-type>
-		<label>Pressure</label>
-		<description>Current Pressure</description>
-		<category>Pressure</category>
-		<state readOnly="true" pattern="%.1f %unit%"/>
-	</channel-type>
-
-	<channel-type id="humidity" advanced="true">
-		<item-type>Number:Dimensionless</item-type>
-		<label>Humidity</label>
-		<description>Current humidity</description>
-		<category>Humidity</category>
-		<state readOnly="true" min="0" max="100" pattern="%.2f %unit%"/>
+	<channel-type id="alert-color" advanced="true">
+		<item-type>Color</item-type>
+		<label>@text/colorChannelLabel</label>
+		<description>@text/colorChannelDescription</description>
+		<state readOnly="true"/>
 	</channel-type>
 
-	<channel-type id="dominentPol">
-		<item-type>String</item-type>
-		<label>Dominent Polutor</label>
-		<state readOnly="true">
-			<options>
-				<option value="pm25">Fine particles</option>
-				<option value="pm10">Coarse dust particles</option>
-				<option value="o3">Ozone</option>
-				<option value="no2">Nitrogen Dioxide</option>
-				<option value="co">Carbon Monoxide</option>
-				<option value="so2">Sulfur Dioxide</option>
-			</options>
-		</state>
+	<channel-type id="alert-icon">
+		<item-type>Image</item-type>
+		<label>@text/pictoChannelLabel</label>
+		<description>@text/pictoChannelDescription</description>
+		<state readOnly="true"/>
 	</channel-type>
 
-	<channel-type id="aqiColor" advanced="true">
-		<item-type>Color</item-type>
-		<label>AQI Color</label>
-		<description>Color associated to given AQI Index.</description>
+	<channel-type id="sensitive-group">
+		<item-type>Switch</item-type>
+		<label>Sensitive group</label>
```

Comment:
Words in labels should be capitalized (except prepositions and so on). See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Reply:
Done